### ISSUE TITLE
 IgxDatePicker accessibility issue

### ISSUE THREAD

**Issue by teodosiah** [IgniteUI/igniteui-angular#8080]
## Description  
Using the following code for igx-datepicker
```
 <igx-date-picker
            [label]="'Date of Birth'"
            name="dateOfBirth"
            [(ngModel)]="generalInfoObj.retailMemberPii.dateOfBirth"
          ></igx-date-picker>
```
Unfortunately generated code fails accessibility testing.

 * igniteui-angular version: 10.1.2

## Expected result  
Input fields should always have a description that is explicitly associated with the field to make sure that users of assistive technologies will also know what the field is for.
If the input field has a visible description indicating the purpose of the field, this description should be explicitly associated to the input field either as an HTML label or using the WAI-ARIA attribute 'aria-labelledby'.
If it is not possible to add a visible description, either add a mouseover text ('title' attribute) to the input field or create an invisible label using the WAI-ARIA attribute 'aria-label'.

**Comment by IvayloG** [comment]
A complete ARIA implementation for this component will be implemented as part of the https://github.com/IgniteUI/igniteui-angular/issues/6346
